recipe/meta.yaml

```diff
@@ -56,7 +58,7 @@ requirements:
     - pyclibrary
     - python
     - setuptools
-    - cuda-version {{ major_version }}
+    - cuda-version {{ major_version }}.{{ minor_version }}
```

Comment:
Made sure there was alignment between CTK versions in `requirements/build` & `requirements/host` by repeating the pattern from `requirements/build`